### ISSUE TITLE
Cannot click allow network button on macOS because it disappears too fast

### ISSUE THREAD

**Issue by kigawas** [abiosoft/colima#162]
Configure firewall and try `colima start`

**Comment by abiosoft** [comment]
Can you explain more? Or is your experience similar to this https://github.com/abiosoft/colima/issues/156?

**Comment by josephschmitt** [comment]
If you _are_ able to accept the dialog quickly enough, the issue is permanently fix. So it's just on initial start the first time you try to start up colima.

**Comment by deviantintegral** [comment]
You can also manually add colima in the "Firewall Options" section of Firewall in Security & Privacy in System Preferences.

I am curious why it needs this in the first place. I would have thought Colima would only need to open an external port when specifically exposing a port to an external interface.

**Comment by requiem240sx** [comment]
I have a use case currently with Colima, wondering if it might be related. 

I'm executing a docker-compose with --env file that calls a remote docker host. It appears like it loads the file just fine, but ignores the docker host portion and executes on my local box.

**Comment by colindean** [comment]
So I delved into this for a problem that I thought existed, but it didn't: because `lima` uses SSH tunnelling to open ports, only ssh should need permission to listen and **it already does have that permission**, AFAIK.

So, something like

```shell
sudo /usr/libexec/ApplicationFirewall/socketfilterfw --add "$(brew --prefix colima)/bin/colima"
```

**should not be necessary** because colima itself doesn't listen. `limactl` _does_ listen along with qemu, obvs, e.g.

```
limactl   40075 colin   15u  IPv4 0xf9381268b43c6a45      0t0  TCP 127.0.0.1:61570 (LISTEN)
qemu-syst 40082 colin   19u  IPv4 0xf9381268b40dc005      0t0  TCP 127.0.0.1:61569 (LISTEN)
```

**Comment by stoutput** [comment]
Yes, very interested in why this is happening. Doesn't seem like incoming network connections should be necessary for colima. See screenshot of popup below (had to record screen it disappeared so fast)
<img width="267" alt="Screen Shot 2022-03-19 at 12 36 16 AM" src="https://user-images.githubusercontent.com/13671824/159108663-30240b2d-bd2c-4fd3-a0bd-7af237e9a487.png">

**Comment by abiosoft** [comment]
Closing as resolved/stale per triage in #1524.
